src/Kconfig

```diff
@@ -37,7 +32,16 @@ config GOLIOTH_OTA_MAX_NUM_COMPONENTS
     help
         Maximum number of components in an OTA manifest.
 
-config GOLIOTH_OTA_OBSERVATION_RETRY_MAX_DELAY_S
+endif # GOLIOTH_OTA
+
+config GOLIOTH_FW_UPDATE
+    bool "Golioth Firmware Update service"
+    select GOLIOTH_OTA
+    help
+        Enable the Golioth Firmware Update service to automatically manage firmware updates
+        utilizing functionality of the Golioth OTA service.
+
+config GOLIOTH_FW_UPDATE_OBSERVATION_RETRY_MAX_DELAY_S
```

Comment:
Should this also be guarded by `if GOLIOTH_FW_UPDATE`?

Reply:
Yes, thanks for catching!